sig-auth/governance.md

```diff
@@ -0,0 +1,150 @@
+# SIG Auth Governance
+
+## Roles
+
+Membership for roles tracked in: [OWNERS]
+
+- **Chairs**
+  - Run operations and processes governing the SIG
+  - Chairs *MUST* remain active in the role and are automatically removed from the position if they
+    are unresponsive for > 3 months and *MAY* be removed if not proactively working with other
+    chairs to fulfill responsibilities.
+  - Chairs *SHOULD* represent a diverse set of organizations.  This is predicated on interest in the
+    role from multiple organizations with qualified candidates.
+  - Chairs *MAY* decide to step down at any time and propose a replacement.  Use lazy consensus
+    amongst chairs with fallback on majority vote to accept proposal.  This *SHOULD* be supported by
+    a majority of SIG Members.
+  - Chairs *MAY* select additional chairs through a [super-majority] vote amongst chairs.  This
+    *SHOULD* be supported by a majority of SIG Members.
+  - Number: 3
+  - Defined in [sigs.yaml]
+
+- **Technical Leads**
+  - Establish new subprojects
+  - Decommission existing subprojects
+  - Seed technical leads nominated by legacy SIG leads from subproject owners
+  - *MUST* ensure that all areas of the SIG (including those that fall outside subprojects) have
+    active owners.
+  - *MAY* set SIG milestone priorities.
+  - *SHOULD* act as an escalation point for technical disputes in subprojects.
+  - *MUST* resolve issues impacting multiple subprojects in the SIG.
+  - *SHOULD* meet or communicate regularly enough to ensure they are aligned. Decisions made by
+    technical leads should be consistent across the SIG. If technical leads are providing
+    inconsistent direction, then they *MUST* align themselves.
+  - Technical Leads *MAY* select additional tech leads through a [super-majority] vote amongst tech
+    leads and chairs.  This *SHOULD* be supported by a majority of SIG Members under
+    [lazy-consensus].
+  - Technical Leads *MUST* remain active in the role and are automatically removed from the position
```

Comment:
might consider rewording this to allow for a short-term planned absence (paternity/maternity leave, sabbatical, etc) with coordination to cover responsibilities. same comment for chairs.